service/collector/helper.go

```diff
@@ -183,3 +175,19 @@ func (h *helper) AWSAccountID(awsClients clientaws.Clients) (string, error) {
 
 	return accountID, nil
 }
+
+// ListReconciledClusters provides a list of clusters reconciled by this
+// particular operator version.
+func (h *helper) ListReconciledClusters() (*infrastructurev1alpha2.AWSClusterList, error) {
```

Comment:
So we go away from caching apprach? Will all collectors get all the objects all the time?

Reply:
Getting 5 objects every minute sounds reasonable to me. @xh3b4sd ?

Reply:
Yes I think this is fine.

Reply:
But isn't that per each collector?

Reply:
Sure. Do you think this is a problem? I was under the impression k8s itself does cashing and doesn't hit Etcd all the time. From that point of view I do not really care. That reflactor caching thing is super ugly somehow and I would like to prevent having that in the code if it does not really solve a problem.

Reply:
I can't tell. Probably doesn't do much performance-wise. It just feels right to me. But that's all I have to say. I won't cry if we don't use it.